src/index.js

```diff
@@ -151,12 +151,7 @@ export { // Moved to math.gl
   SphericalCoordinates
 } from 'math.gl';
```

Comment:
At least we don't need to export experimental math.gl classes like SphericalCoordinates.